src/components/VendorSalesPage/Location.js

```diff
@@ -4,6 +4,17 @@ import 'antd/dist/antd.css';
 import { connect } from 'react-redux';
 
 class Location extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Why is this empty?

Reply:
The state is empty because afterwards I am using setstate and setting the key and value both